CONTRIBUTING.md

```diff
@@ -1,86 +1,187 @@
+-   [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/)
+-   [Building Welcoming Communities](https://opensource.guide/building-community/)
 
-# Contributing
+## Get Involved
 
-1. Fork it
-2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Push to the branch (`git push origin my-new-feature`)
-5. Create new Pull Request
+There are many ways to contribute to Pact Python and the broader Pact ecosystem. Here's a few ideas to get started:
 
-If you are intending to implement a fairly large feature we'd appreciate if you open
-an issue with GitHub detailing your use case and intended solution to discuss how it
-might impact other work that is in flight.
+-   Look through the [open issues](https://github.com/pact-foundation/pact-python/issues). Provide workarounds, ask for clarification, or suggest labels. Help [triage issues](#triaging-issues-and-pull-requests).
+-   If you find an issue you would like to fix, [open a pull request](#pull-requests). Issues tagged as [_Good first issue_](https://github.com/pact-foundation/pact-python/labels/Good%20first%20issue) are a good place to get started.
+-   Read through the [docs](https://docs.pact.io). If you find anything that is confusing or can be improved, you can click "Edit this page" at the bottom of most docs, which takes you to the GitHub interface to make and propose changes.
+-   Take a look at the [features requested](https://github.com/pact-foundation/pact-python/labels/feature) by others in the community and consider opening a pull request if you see something you want to work on.
 
-We also appreciate it if you take the time to update and write tests for any changes
-you submit.
+Contributions are very welcome. If you think you need help planning your contribution, please ping us on [Slack](https://slack.pact.io) and let us know you are looking for a bit of help.
 
-[examples]: https://github.com/pact-foundation/pact-python/tree/master/examples
+### Join our Slack
 
-## Commit messages
+We have a [Slack](https://slack.pact.io) to discuss all things about Pact and its development. Feel free to ask questions about Pact Python specifically in the [`#pact-python`](https://pact-foundation.slack.com/archives/C9VECUP6E) channel, or broader questions about the Pact ecosystem over in the [`#general`](https://pact-foundation.slack.com/archives/C5F4KFKR8) channel.
 
-pact-python is adopting the [Conventional Commits](https://www.conventionalcommits.org)
-convention. Please ensure you follow the guidelines, we don't want to be that person,
-but the commit messages are very important to the automation of our release process.
+### Triaging Issues and Pull Requests
 
-Take a look at the git history (`git log`) to get the gist of it.
+One great way you can contribute to the project without writing any code is to help triage issues and pull requests as they come in.
```

Comment:
nice, there is a new pact triage board

https://github.com/orgs/pact-foundation/projects/16

maybe you could link to that and provide a filter on pact-python things

Reply:
Done :)

Reply:
>At the moment, it is a bit overwhelming seeing a giant list in one column, and hardly anything else anywhere else. What do you think?

It is yes, it was a juggle between do we just bring in new ones and leave old ones to stagnate, having them in the list means anyone can dip in and pick something to work on.

Also I'm not very meticulous in moving tickets across when I am working on them, I might jump into to something because of GH notification elsewhere and then remember that its being tracked on the triage board. It's only been introduced within the last month, so still need to get it into muscle memory

Reply:
> On second thought, we probably need to get out boards up to scratch first and make sure they are actively being used. At the moment, it is a bit overwhelming seeing a giant list in one column, and hardly anything else anywhere else. What do you think?

I think we leave it out for now and when Yousaf and I have got that board into shape, we can do an organisation-wide review to ensure it's communicated effectively.